doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -51,6 +51,14 @@ GNU ARM Embedded
       C:\gnu_arm_embedded
 
 
+   .. warning::
+
+      On macOS, if you are having trouble with the suggested procedure, there is an unofficial package on brew that migth help you.
+      Run ``brew install gcc-arm-embedded`` and configure the variables
+
+      - Set :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``gnuarmemb``.
+      - Set :envvar:`GNUARMEMB_TOOLCHAIN_PATH` to the brew installation directory (something like ``/usr/local``)
```

Comment:
I think this might not be true on Apple Silicon based machines, where homebrew binaries are placed in ``opt/`` if I am not mistaken.

Reply:
I am using MacPorts (https://www.macports.org/) and it puts stuff in  /opt/local/.

Reply:
Thanks for helping!
As far as I know, all the binaries in `opt` and `bin` are symlinked, the original file are on the cellar folder.
![image](https://user-images.githubusercontent.com/530406/109797292-902fa400-7c19-11eb-8dba-0966042e57a0.png)